packages/binutils/binutils.spec

```diff
@@ -31,7 +31,6 @@ Requires: %{name}
   --disable-gdb \
   --with-system-zlib \
   --without-gnu-as \
-  --disable-plugins \
```

Comment:
I was unaware of libdep before this change, but it seems the libdep plugin only was introduced with binutils 2.36. It is not included in the current/to be replaced version of binutils. With this, I don't understand the intention of the change/commit message. Is the libdep plugin built independently of the choice to `--enable-plugins`/`--disable-plugins`?

It also seems the libdep plugin is useful when linking against static libraries, since transitive dependencies can be pulled in without having to explicitly specify them. This doesn't seem too useful on a Bottlerocket host. Is this just to more closely mirror the version of binutils there is in the SDK?

Reply:
I agree that my commit message could be more clear on what motivated the change.

The upstream configure script no longer has an option to disable building plugins, likely because libdep was added in a way that causes it to always be built.

I could try to patch it out but opted to just go with it. I don't think it will be useful in this project, but it also seems harmless to have around.

Reply:
At least IIRC the configure option is gone. But either way it has no effect on libdep being built.

Reply:
Agree it's no big deal to ship libdep. Thanks for clarifying your reasoning!